src/features/for fun/snipe.command.ts

```diff
@@ -0,0 +1,58 @@
+import {
+    EmbedBuilder,
+    PermissionFlagsBits,
+    SlashCommandBuilder,
+    inlineCode,
+    type ChatInputCommandInteraction,
+    type Role,
+  } from "discord.js";
+  
+  import { SlashCommandHandler } from "../../abc/command.abc";
+  import {
+    isMissingAccessError,
+    isMissingPermissionsError,
+  } from "../../utils/errors.utils";
+  import {
+    OFFICERS_ROLE_ID
+  } from "../../utils/snowflakes.utils";
+  class SnipeCommand extends SlashCommandHandler {
+    public override readonly definition = new SlashCommandBuilder()
+      .setName("snipe")
+      .setDescription("Upload a snipe of ")
+      .setDefaultMemberPermissions(PermissionFlagsBits.ManageMessages)
+      .toJSON();
+  
+    public override async execute(
+      interaction: ChatInputCommandInteraction,
+    ): Promise<void> {
+      const { guild, channel } = interaction;
+      if (!guild || !channel) {
+        await interaction.reply(
+          "⚠️ This command can only be used within a text channel.",
+        );
+        return;
+      }
+  
+      if (!channel.isTextBased()) {
+        await interaction.reply(
+          "⚠️ This command can only be used in text channels.",
+        );
+        return;
+      }
+  
+      const lastMessage = channel.messages.cache.last();
+      if (!lastMessage) {
+        await interaction.reply("⚠️ No messages to snipe.");
+        return;
+      }
+  
+      const embed = new EmbedBuilder()
+        .setColor("#FF0000")
```

Comment:
If your intent is to use a red color, use the pre-made constant from the discord.js library:

```suggestion
        .setColor(Colors.Red)
```

This isn't *exactly* `#FF0000`, but close enough, and it keeps things simpler. You can also play around with the other provided colors, like `Colors.DarkRed`.